symplyphysics/laws/chemistry/distance_of_greatest_convergence_of_particles_in_magnetron.py

```diff
@@ -0,0 +1,38 @@
+from sympy import Eq, solve, log
+from symplyphysics import (units, Quantity, Symbol, validate_input,
+    validate_output, dimensionless,)
+
+# Description
+## The traveling atom moves towards the substrate in the magnetron. At the same time, it collides with gas atoms.
+## The energy transfer coefficient in these collisions depends on the mass of the traveling atom and the mass of the gas atom.
+## The distance of the greatest convergence of colliding particles can be calculated using the model of quasi-rigid spheres.
+
+## Law is: r = -(0.122e-10 * 2 * Z^0.0387) * ln(U / (95.863 * Z^1.4766 * (Z^2)^0.75), where
+## r - the distance of the greatest convergence of two colliding particles,
+## U - discharge voltage,
+## Z - the number of the gas atom.
+
+distance_of_convergence_of_particles = Symbol("distance_of_convergence_of_particles", units.length)
+
+discharge_voltage = Symbol("discharge_voltage", units.voltage)
+number_of_gas_atom = Symbol("number_of_gas_atom", dimensionless)
+
+first_constant = Quantity(0.122e-10 * units.meter)
+second_constant = Quantity(95.863 * units.volt)
+expression_1 = first_constant * 2 * number_of_gas_atom**0.0387
+expression_2 = second_constant * number_of_gas_atom**1.4766
+
+law = Eq(distance_of_convergence_of_particles, -expression_1 * log(discharge_voltage / (expression_2 * (number_of_gas_atom**2)**0.75)))
```

Comment:
Why `(number_of_gas_atom**2)**0.75` is not simplified? I guess it should be `number_of_gas_atom**1.5`.

Reply:
> Why `(number_of_gas_atom**2)**0.75` is not simplified? I guess it should be `number_of_gas_atom**1.5`.

This is how I wrote down the formula, because in general sources it is written that way. But yes, it would be logical to shorten it. Shorten it?

Reply:
Yes. But make sure there is no typo in law. Maybe such power was for a reason.